docs/generated/packages/cypress/executors/cypress.json

```diff
@@ -24,12 +24,14 @@
         "type": "string",
         "description": "The path of the Cypress configuration json file.",
         "x-completion-type": "file",
-        "x-completion-glob": "cypress?(*)@(.js|.ts|.json)"
+        "x-completion-glob": "cypress?(*)@(.js|.ts|.json)",
+        "x-priority": "important"
```

Comment:
Don't mark `required` items as `important` since they will automatically be categorized as such. It would just create duplicate maintenance overhead